src/environmentd/src/http.rs

```diff
@@ -76,6 +79,7 @@ pub struct HttpConfig {
     pub adapter_client: mz_adapter::Client,
     pub allowed_origin: AllowOrigin,
     pub active_connection_count: Arc<Mutex<ConnectionCounter>>,
+    pub concurrent_webhook_req_count: Option<usize>,
```

Comment:
This is totally subjective so feel free to disagree. You could modify this to be just a `usize` and do the `.unwrap_or(webhook::CONCURRENCY_LIMIT)` logic when this object is constructed. A `None` value here might be misinterpreted as no limit instead of default limit.

Reply:
That makes sense to me, updated!